Aboutme.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+var answer = prompt('Do you like scary stories?');
+if (answer.toLowerCase() === 'yes' || answer.toLowerCase() === 'y') {
+  alert('BOO! (it\'s a short story)');
+} else if (answer.toLowerCase() === 'no' || answer.toLowerCase() === 'n') {
+  alert('It\'s your loss.');
+} else {
+  alert('No need to get scared and start typing out control');
+}
+
+var answer = prompt('Are you ready to party?');
```

Comment:
You are redeclaring the answer variable for each question.  Consider using a different variable with a descriptive name for each question, that way you retain the ability to access those variables later.  For example, you could use `var answerOne`, `var answerTwo`, etc.